tasks/electron.js

```diff
@@ -73,7 +73,8 @@ function _processZipEntry(zip, dest, entry) {
   }
 }
 
-function _zipExtract(zipfile, dest = './') {
+function _zipExtract(zipfile, dest) {
+  dest = (typeof dest !== 'undefined') ? dest : './';
```

Comment:
You shouldn't be installing node through Debians package manager, but rather with [nvm](https://github.com/creationix/nvm). Babel is using the node 6 preset, so babel setup is working as intended and not converting syntax that node 6 supports.